qa/acceptance/Vagrantfile

```diff
@@ -0,0 +1,64 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+
+  config.vm.define "ubuntu-1204" do |ls|
+    ls.vm.box = "elastic/ubuntu-12.04-x86_64"
+    ubuntu_common_config(ls)
+  end
+
+  config.vm.define "ubuntu-1404" do |ls|
+    ls.vm.box = "elastic/ubuntu-14.04-x86_64"
+    ubuntu_common_config(ls)
+  end
+
+  config.vm.define "centos-6" do |ls|
+    ls.vm.box = "elastic/centos-6-x86_64"
+    centos_common_config(ls)
+  end
+
+  config.vm.define "centos-7" do |ls|
+    ls.vm.box = "elastic/centos-7-x86_64"
+    centos_common_config(ls)
+  end
+
+end
+
+def centos_common_config(ls)
+  ls.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 4
+  end
+ls.vm.synced_folder "../../build", "/logstash-build"
+  ls.vm.provision :shell do |sh|
+    sh.path = "sys/centos/bootstrap.sh"
+    sh.privileged = true
+  end
+
+  ls.vm.provision :shell do |sh|
+    sh.path = "sys/centos/user_bootstrap.sh"
+    sh.privileged = false
+  end
+
+  #ls.vm.network "forwarded_port", guest: 9600, host: 9601
+end
+
+def ubuntu_common_config(ls)
+  ls.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 4
+  end
+  ls.vm.synced_folder "../../build", "/logstash-build"
+  ls.vm.provision :shell do |sh|
+    sh.path = "sys/ubuntu/bootstrap.sh"
+    sh.privileged = true
+  end
+
+  ls.vm.provision :shell do |sh|
+    sh.path = "sys/ubuntu/user_bootstrap.sh"
+    sh.privileged = false
+  end
+
+  #ls.vm.network "forwarded_port", guest: 9600, host: 9600
+end
```

Comment:
It for sure makes the configuration shorter for now, my thoughts here are that for now it simplify, as we have nothing platform specific, but if you take a look at https://github.com/elastic/elasticsearch/blob/master/Vagrantfile , the source of this file it gets some thing that are platform specific, so for now I stayed to something like this, but I'm good to make it as DRY as possible.